src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { Button, Nav, Navbar } from 'react-bootstrap';
+
+import { Profile } from '../App';
+import LoginButton from '../LoginButton';
+import styles from './styles.module.scss';
+
+type Props = {
+  onLogOut: () => void;
+  profile: null | Profile;
+};
+
+export class NavbarBase extends React.Component<Props> {
+  render() {
+    const { onLogOut, profile } = this.props;
+
+    return (
+      <Navbar bg="dark" className={styles.Navbar} expand="lg" variant="dark">
+        <Navbar.Brand href="#">addons-code-manager</Navbar.Brand>
+        <Nav>
+          {profile ? (
+            <Nav.Link className="Navbar-username">{profile.name}</Nav.Link>
+          ) : null}
+          {profile ? (
+            <Button className="Navbar-logOut" onClick={onLogOut}>
+              Log out
```

Comment:
Can we defer the i18n stubs in a new issue? I thought we wanted to give a more react-y API a try (Lingui?), so this could end up not being a `gettext` function.

Reply:
I'm fine with either. I suppose it's not going to be too big a deal to find and replace all instances of `gettext` with something else if/when we decide to go in a different direction, but I do agree that switching from a function to a tag (Lingui style), would be a bit more work.

I think we believe that we probably will not internationalize this app at all, in which case whatever is the least amount of work at this point is probably what we should do, and I think that's probably using a function.

I'll wait to hear back from both of you before proceeding with this.